src/app/file-manager/models/entity-status.ts

```diff
@@ -0,0 +1,13 @@
+export class EntityStatus {
+
+  static readonly PENDING = 0;
+  static readonly INPROGRESS = 1;
+  static readonly PAUSED = 2;
+  static readonly COMPLETED = 3;
+  static readonly CANCELED = 4;
+  static readonly FAILED = 5;
+
+  static lenght = 6;
```

Comment:
Typo `length`? It doesn't appear to be used anywhere, can it be removed?

Reply:
Yes, it is a typo and will be removed.